packages/vue-client/src/components/boards/MulticolorGraphBoard.vue

```diff
@@ -95,16 +95,16 @@ const viewBox = computed(() => {
       />
     </g>
     <g>
-      <IntersectionAnnotation
-        v-for="(intersection, index) in intersections.filter(
-          (_, idx) => props.board.at(idx)?.annotation,
-        )"
-        :key="index"
-        :cx="intersection.position.X"
-        :cy="intersection.position.Y"
-        :r="0.48"
-        :annotation="props.board.at(index)?.annotation!"
-      />
+      <template v-for="(intersection, index) in intersections">
+        <IntersectionAnnotation
+          v-if="props.board.at(index)?.annotation"
```

Comment:
Does this change do anything?  IIUC it's equivalent to the `filter`

Just wondering in case the same code in `MulticolorGridBoard` needs to be fixed too.

Reply:
Yes this is a bugfix. It's not equivalent because after filtering the array, the indices don't correspond to the indices of the original array any more. This made it so that `undefined` was passed as prop for `annotation`.

Reply:
Oh makes sense! Grid.filter can preserve index but Array.filter cannot